Droidcon-Boston/app/src/main/res/layout/quoted_tweet_item.xml

```diff
@@ -25,6 +26,10 @@
         android:id="@+id/quotedName"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
+        android:maxLines="1"
+        android:ellipsize="end"
+        app:autoSizeTextType="uniform"
+        app:autoSizeMinTextSize="10sp"
```

Comment:
if we use a 12sp minimum, does it have a big impact on when the text gets truncated?  I ask, because 12 I believe is the recommended minimum.

It was still looking pretty readable at 10 though so if you feel like 12sp starts to wrap too much let's stick with 10sp.  I'm happy to leave that to your best judgement